client/src/pages/Login/Login.jsx

```diff
@@ -1,9 +1,11 @@
-import React from 'react'
+import React from "react";
 
 export const Login = () => {
   return (
     <div>
-        <h1 className="border-2 border-rose-600 text-blue-600">Login</h1>
+      <h1 className="border-2 border-rose-600 text-blue-600">Login</h1>
+      <h2>holi</h2>
+      <h2>hello</h2>
```

Comment:
```suggestion
      <h2>hola mundo</h2>
```
de debe poner en esoañol todo